example/server.js

```diff
@@ -2,13 +2,17 @@
 
 var Hapi = require('hapi');
 var Hoek = require('hoek');
+var fs = require('fs');
 var tags = require('../lib/index.js');
-
+var tagsData = JSON.parse(fs.readFileSync('./example/tags.json').toString());
```

Comment:
Ok, we had a problem with caching in the past using `require`, so thought this would be a safer way of doing it. But happy to change back.